titus-ext/runner/titusgateway.properties

```diff
@@ -1,2 +1 @@
-titus.gateway.masterIp=${TITUS_MASTER_HOST}
-titus.gateway.masterHttpPort=7001
\ No newline at end of file
+titus.masterClient.masterIp=${TITUS_MASTER_HOST}
```

Comment:
If this is the correct way, should the guide on https://netflix.github.io/titus/install/master/ section Gateway also be updated? It currently displays 
```
Create ~/titusgateway.properties with the properties:


titus.gateway.masterIp=<MASTER_IP>
titus.gateway.masterHttpPort=<MASTER_PORT>
```

Reply:
yes, you are right. I will get it fixed.